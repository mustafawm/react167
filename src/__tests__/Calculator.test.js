import React from 'react';
import {render} from 'react-testing-library';
import CalculatorDisplay from '../components/Calculator';

test('Calc display mounts', () => {
  const {container} = render(<CalculatorDisplay value="0" />);
  expect(container).toMatchSnapshot();
});
