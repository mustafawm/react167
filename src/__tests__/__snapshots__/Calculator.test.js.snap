// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Calc display mounts 1`] = `
<div>
  <div
    class="calculatorDisplay"
  >
    <div
      class="autoScalingText"
      style="transform: scale(1,1);"
    >
      0
    </div>
  </div>
</div>
`;
